pom.xml

```diff
@@ -78,7 +78,7 @@
 	</mailingLists>
 	<properties>
 		<!-- metaschema dependencies -->
-		<dependency.metaschema-framework.version>0.12.0</dependency.metaschema-framework.version>
+		<dependency.metaschema-framework.version>0.12.1</dependency.metaschema-framework.version>
 		<dependency.liboscal-java.version>3.0.1-SNAPSHOT</dependency.liboscal-java.version>
```

Comment:
@aj-stein-nist You will first need to release and update the liboscal-java version.

Reply:
Good point, I should have waited for moving this up, thanks for the reminder and answering when I asked last night in Element.

Reply:
Done in touched-up commit https://github.com/usnistgov/oscal-cli/commit/557e6a9e5cdc2491a47df36b447fa5daefd26e14.